L1Trigger/TrackTrigger/src/TTStubAlgorithm_official.cc

```diff
@@ -16,16 +16,18 @@ void TTStubAlgorithm_official<Ref_Phase2TrackerDigi_>::PatternHitCorrelation(
     bool& aConfirmation,
     int& aDisplacement,
     int& anOffset,
-    float& anROffset,
     float& anHardBend,
-    const TTStub<Ref_Phase2TrackerDigi_>& aTTStub) const {
+    const TTStub<Ref_Phase2TrackerDigi_>& aTTStub) const
+// Removed real offset.  Ivan Reid 10/2019
```

Comment:
this looks either spurious or superfluous

Reply:
fixed.